### PR TITLE
Updates based on changes in wsgi-base

### PR DESCRIPTION
**Pull request by kvikshaug**

### CODE REVIEW

File: alpine/Dockerfile [hunk 1]
```diff
@@ -18,12 +18,19 @@ WORKDIR /opt
...
-# `g++` is required for building `psycopg2` but all build dependencies are
-# later removed again to reduce the layer size.
 RUN set -eux \
-    && apk add --no-cache --virtual .build-deps postgresql-dev g++ \
+    # Install build dependencies to make sure we can build packages that don't
+    # provide pre-built wheels from source.
+    && apk add --no-cache --virtual .build-deps g++ \
+    # Install dev libraries for psycopg2. Note that these are needed at runtime.
+    && apk add --no-cache postgresql-dev python3-dev \
+    # Compile the dependencies for base + SQL.
```
Comment (Midnighter) [inline alpine/Dockerfile:27]: I would structure it in the following way:

```dockerfile
    && apk add --no-cache --virtual .build-deps g++ postgresql-dev \
    && apk add --no-cache postgresql \
```

We could also update the index once and later remove it but fetching seems pretty fast.

```dockerfile
&& apk update
...
&& rm -rf /var/cache/apk/*
```

Comment (Midnighter) [reply alpine/Dockerfile:27]: The difference in size between including the headers or not is 13 MB btw.

File: alpine/sql-requirements.in [hunk 2]
```diff
@@ -1,7 +1,13 @@
--r wsgi-requirements.txt  # Maintain dependencies of the base image.
+# Maintain exact dependencies of the base image.
+-r base-requirements.txt
```
Comment (Midnighter) [inline alpine/sql-requirements.in]: I'm wondering if the absolute path should be preferred? That way the pip-sync can be executed from anywhere in child images without problems.

```suggestion
-r /opt/base-requirements.txt
```

**Review by Midnighter — CHANGES_REQUESTED** [review]
Nice, I think these are really good changes :smiley: I'm also happy that I'm still away and don't have to adjust our services, though :grimacing:

File: alpine/Dockerfile (hunk 1 above)
Comment (kvikshaug) [reply alpine/Dockerfile:27]: Ok, thank you - the 13 MB are now shaved off.

**Review by Midnighter — APPROVED** [review]
:+1:

**Pull request merged**